scripts/engine/component.py

```diff
@@ -206,6 +206,15 @@ def __init__(self, active: List[Affliction] = None):
     def add(self, affliction: Affliction):
         self.active.append(affliction)
 
+    def remove(self, affliction: Affliction):
```

Comment:
It's better for the class Afflictions to know how it's implemented in the inside instead of world.py having that information